src/e-init.js

```diff
@@ -126,6 +127,7 @@ program
   .option('--tsan', `When building, enable clang's thread sanitizer`, false)
   .option('--msan', `When building, enable clang's memory sanitizer`, false)
   .option('--lsan', `When building, enable clang's leak sanitizer`, false)
+  .option('--mas', `Build for the macOS App Store`, false)
```

Comment:
```suggestion
  .option('--mas', 'Build for the macOS App Store', false)
```
Only because you made the same change to the other line. 😅